### DISCUSSION TITLE
Feature request: Possibility to use Tabs with CSS only

### DISCUSSION THREAD

**Discussion by w3teal** [beercss/beercss#405] [Ideas]
Lately, i really like to explore more possibility with CSS hacks like `checkbox` and `radio`. Like in Dialogs i made in #404, it's using checkbox hack. Meanwhile this one, i use radio hack for the CSS-only tabs which is inspired from this codepen (https://codepen.io/makzan/pen/MWYMVoM).

My codepen also supporting helpers like `min`, `vertical` for the `label`s and transition for the page like `top`, `bottom`, `left` and `right`. This implementation is almost better than any alternatives which use absolute positioning or `nth-child`.
Demo: https://codepen.io/LIGMATV/pen/dPbVRGZ
And also, for Navigation tabbed: https://codepen.io/LIGMATV/pen/jENGwJr

**Comment by laurentpayot** [comment]
@LIGMATV Once again, great CSS-only trick! :+1: 

![cssonlymore](https://github.com/user-attachments/assets/1afbdfd4-2e92-43d1-9501-a42b5ec71fc3)

**Comment by w3teal** [comment]
I think this also has [a problem with semantic](https://github.com/beercss/beercss/discussions/404#discussioncomment-11761024), maybe it would be better to use [`details` `name`s](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#multiple_named_disclosure_boxes), considering it's also [supported in average browsers.](https://caniuse.com/mdn-html_elements_details_name) I will like to demoing for this one 👀
